### DISCUSSION TITLE
How to compile tutorial

### DISCUSSION THREAD

**Discussion by laucer** [amzn/selling-partner-api-models#3259] [General]
Guys, I think we need to create a simple tutorial, how to compile the code to produce a nice JAR library. 

Currently when following this tutorial: https://jesseevers.com/spapi-first-application/
after ./gradlew build 
I get huge amounts of errors.

Please spend a while and write a clean documentation

**Comment by delevko** [comment]
Guys! I'm facing the same issue. The updated library doesn't compile.
Please, help

**Comment by JasiekLewandowski** [comment]
I thought I was the only one experiencing that. Compiling code with new lib is a mess!  
Please help!
